cli/tests/debug_with_log_level_and_RUST_LOG.out

```diff
@@ -0,0 +1,25 @@
+Deno isolate init with snapshots.
+rust:shared_queue:reset
+main_module file://[WILDCARD]/cli/tests/002_hello.ts
+fetch_source_file specifier: file://[WILDCARD]/cli/tests/002_hello.ts maybe_referrer: None
+⚠️️  Granted read access to "[WILDCARD]002_hello.ts"
+Check file://[WILDCARD]/cli/tests/002_hello.ts
+Deno isolate init with snapshots.
+rust:shared_queue:reset
+⚠️️  Granted access to high precision time
+⚠️️  Granted access to high precision time
+DEBUG - Compilation Statistics:
+Files: [WILDCARD]
+Nodes: [WILDCARD]
+Identifiers: [WILDCARD]
+Symbols: [WILDCARD]
+Types: [WILDCARD]
+Instantiations: [WILDCARD]
+Parse time: [WILDCARD]
+Bind time: [WILDCARD]
+Check time: [WILDCARD]
+Emit time: [WILDCARD]
+Total TS time: [WILDCARD]
+Compile time: [WILDCARD]
+compiled filename: "[WILDCARD]002_hello.ts.js"
+register_complete file://[WILDCARD]/cli/tests/002_hello.ts
```

Comment:
These tests are too specific. We'll have to update the every time we change a debug output. We just want something that checks the logging produces some output - not a specific output. 

I will fix this up. Thanks for the good start @tokiedokie